recipe/meta.yaml

```diff
@@ -23,7 +23,6 @@ build:
 requirements:
   build:
     - {{ compiler('cxx') }}
-    - {{ stdlib("c") }}
```

Comment:
The rerender seems to have removed this again (probably due to timing issues between follow-up commits and a previous rerender)